File: articles/online-algorithms-and-decisions-under-arrival/java/src/main/java/org/contentcatalyst/algorithms/OnlineDecisionAudit.java
package org.contentcatalyst.algorithms;
public class OnlineDecisionAudit {
  public static void main(String[] args){
    double threshold = 5 * 10.0 + 50.0;
    double offline = 50.0;
    System.out.println("test_name,value");
    System.out.println("threshold_strategy," + threshold);
    System.out.println("offline_optimum," + offline);
    System.out.println("ratio," + (threshold / offline));
  }
}

File: articles/online-algorithms-and-decisions-under-arrival/c/online_decision_audit.c
#include <stdio.h>
int main(void){ double threshold=5*10.0+50.0, offline=50.0; printf("test_name,value\nthreshold_strategy,%.3f\noffline_optimum,%.3f\nratio,%.3f\n", threshold, offline, threshold/offline); return 0; }
